src/IntroductionComponent.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './App.scss';
+
+
+export interface IntroductionProps {
+    divClassName: string,
+    imgSrc: string,
+    imgAlt: string,
+    imgClassName: string,
+    h1Contents: string,
+    p1Contents: string,
+    p2Contents: string
+    href: string,
+    hrefContents: string,
+    hrefClassName: string
+}
+
+export function IntroductionComponent(props: IntroductionProps) {
+    return (
+        <div className={props.divClassName}>
+            <img src={props.imgSrc} className={props.imgClassName} alt={props.imgAlt} />
+            <h1> {props.h1Contents} </h1>
```

Comment:
Though browsers will usually trim the surrounding whitespace, we try to write code not to rely on it so that, when reading the HTML, it's easier to translate it in our heads into what it would look like.